bundles/org.openhab.binding.boschshc/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -17,19 +17,19 @@
                 <required>true</required>
```

Comment:
Would it be possible to discover the IP address automatically? If not we could add a link to the official Bosch support page which describes how to find out the IP: https://www.bosch-smarthome.com/de/de/service/bei-der-erstinstallation-werde-ich-nach-der-ip-adresse-des-shc-gefragt

We could ask in the Bosch GitHub repository (https://github.com/BoschSmartHome/bosch-shc-api-docs) how an automatic discovery could be done. The official Bosch Smart Home app does it as well, so it might be possible: https://tt-smarthome.resource.bosch.com/documents/Manual/UK/System_Solutions/Controller_EN_161109_web.pdf

Reply:
The SHC returns [public information](https://github.com/BoschSmartHome/bosch-shc-api-docs/tree/master/postman#check-the-connection-with-the-bosch-smart-home-controller-in-your-local-network) without pairing.
If we assume that both - SHC and OpenHAB - have IPs in the same network it should be possible find with a simple try & error approach.

I suggest to add/implement this later, after this PR, via a SHC bridge discovery service.

Reply:
That would be a good fallback if there is no official way. And yes, let's do it in a new issue, I will create one with the idea.

Reply:
https://github.com/stefan-kaestle/openhab2-addons/issues/13